app/models/grant_reviewer/invitation.rb

```diff
@@ -0,0 +1,29 @@
+class GrantReviewer::Invitation < ApplicationRecord
+  has_paper_trail versions: { class_name: 'PaperTrail::GrantReviewerInvitationVersion'},
+                  meta:     { grant_id: :grant_id, email: :email }
+
+  belongs_to :grant
+  belongs_to :inviter,  class_name: 'User',
+                        foreign_key: :invited_by_id
+  belongs_to :invitee,  class_name: 'User',
```

Comment:
why optional?

Reply:
Invitee's email isn't in the db and is being invited to log in. Once they log in, their id be added